package.json

```diff
@@ -17,9 +17,8 @@
     "jasmine-jquery": "^2.1.1",
     "jquery": "^2.2.4",
     "jshint": "^2.9.5",
-    "node-sass": "^4.14.1",
-    "sass-loader": "^6.0.6",
-    "style-loader": "^0.18.2",
-    "webpack": "^3.6.0"
```

Comment:
Are we able to remove `webpack` here? For example, we have a `webpack.config.js` that I believe is run during `make static`? Given this, I'm actually surprised our CI doesn't fail without `webpack` installed. Are we actually attempting to compile a production build of the static assets during CI?

Reply:
added it back. i had removed to dealing with a dependency conflict. i am looking into the ci tests now.